static/scss/email-composition-dialog.scss

```diff
@@ -1,67 +1,133 @@
-.email-composition-dialog {
-  .email-composition-contents {
-    display: flex;
-    flex-direction: column;
-
-    .subheading-section {
-      padding-bottom: 20px;
+.email-composition-dialog-wrapper {
+  overflow: auto;
 
-      .subheading-to {
-        padding: 0 10px;
+  .email-composition-dialog {
+    padding-top: 0 !important;
 
+    > div {
+      &:first-child {
```

Comment:
the selector below is `> div:first-child > div`, which is sort of abstract - I would suggest trying to figure out a way to refactor that to use a simple class, which would be more readable in the future.